cellprofiler/modules/enhanceedges.py

```diff
@@ -47,87 +50,110 @@ class EnhanceEdges(cpm.Module):
-            the maximum as the pixel's value.</li>
-            </ul>''' % globals())
+                [M_SOBEL, M_PREWITT, M_ROBERTS, M_LOG, M_CANNY, M_KIRSCH], doc='''\
+There are several methods that can be used to enhance edges:
+
+-  *%(M_SOBEL)s:* Finds edges using the %(M_SOBEL)s approximation to
+   the derivative. The %(M_SOBEL)s method derives a horizontal and
+   vertical gradient measure and returns the square-root of the sum of
+   the two squared signals.
+-  *%(M_PREWITT)s:* Finds edges using the %(M_PREWITT)s approximation
+   to the derivative. It returns edges at those points where the
+   gradient of the image is maximum.
+-  *%(M_ROBERTS)s:* Finds edges using the Roberts approximation to the
+   derivative. The %(M_ROBERTS)s method looks for gradients in the
+   diagonal and anti-diagonal directions and returns the square-root of
+   the sum of the two squared signals. This method is fast, but it
+   creates diagonal artifacts that may need to be removed by smoothing.
+-  *%(M_LOG)s:* Applies a Laplacian of Gaussian filter to the image and
+   finds zero crossings.
+-  *%(M_CANNY)s:* Finds edges by looking for local maxima of the
+   gradient of the image. The gradient is calculated using the
+   derivative of a Gaussian filter. The method uses two thresholds to
+   detect strong and weak edges, and includes the weak edges in the
+   output only if they are connected to strong edges. This method is
+   therefore less likely than the others to be fooled by noise, and more
+   likely to detect true weak edges.
+-  *%(M_KIRSCH)s:* Finds edges by calculating the gradient among the 8
+   compass points (North, North-east, etc.) and selecting the maximum as
+   the pixel’s value.
+''' % globals())
 
         self.direction = cps.Choice(
                 "Select edge direction to enhance",
-                [E_ALL, E_HORIZONTAL, E_VERTICAL], doc='''
-            <i>(Used only with %(M_PREWITT)s and %(M_SOBEL)s methods)</i> <br>
-            The direction of the edges
-            are you are identifying in the image (predominantly horizontal, predominantly vertical,
-            or both).''' % globals())
+                [E_ALL, E_HORIZONTAL, E_VERTICAL], doc='''\
+*(Used only with "%(M_PREWITT)s" and "%(M_SOBEL)s" methods)*
 
-        self.wants_automatic_sigma = cps.Binary("Calculate Gaussian's sigma automatically?", True)
+The direction of the edges are you are identifying in the image
+(predominantly horizontal, predominantly vertical, or both).
+''' % globals())
 
-        self.sigma = cps.Float("Gaussian's sigma value", 10)
+        self.wants_automatic_sigma = cps.Binary("Calculate Gaussian's sigma automatically?", True, doc="""\
+Select *%(YES)s* to automatically calculate the Gaussian's sigma.
+
+Select *%(NO)s* to manually enter the value.
+        """)
+
+        self.sigma = cps.Float("Gaussian's sigma value", 10, doc="""Set a value for Gaussian's sigma""")
```

Comment:
Add ` % globals()` after the end of the docstring (but before the parenthesis) to get the variables to render properly.